pkg/arrai/json.arrai

```diff
@@ -0,0 +1,48 @@
+let model = //{./sysl}.normalize(//{./sysl}.load('out/model.pb'));
+
+# Transforms the model's information about the named package to a JSON content.
+let toPackagePage = \m \appPackage
+    let apps = (m.apps where .appPackage = appPackage);
+    let endpoints = apps <&> m.endpoints;
+
+    $`
+    {
+      "apps": {
+        ${apps => $`
+              "${.appName}": {
+                "name": "${.appName}",
+                "description": "${//seq.sub('\n', '\\n', .appDescription)}",
+                "package": "${appPackage}",
+                "endpoints":  {
+                  ${.appEndpoints => $`
```

Comment:
Ah, I see. Yeah, at this point you wanna do the join:
```suggestion
                  ${. <&> m.endpoints => $`
```
That'll give you a set of endpoints belonging to the current app.